src/Photos.vue

```diff
@@ -82,6 +82,10 @@ export default {
 	flex-grow: 1;
 	flex-direction: column;
 	align-content: space-between;
+
+	&.icon-loading::after {
+		margin-top: 16px;
```

Comment:
It's probably safer to make sure the container have a min height of 100%? :)

Reply:
What you mean ?

Reply:
That the icon loading is centered on the parent space.
The real issue is that the parent (the app container) should have a minimum height of 100%

Reply:
Oh got it!